.github/workflows/terraform.yml

```diff
@@ -10,12 +10,27 @@ on:
   pull_request:
 
 jobs:
+  determine_modules:
+    name: "Discover Modules"
+    runs-on: ubuntu-latest
+    outputs:
+      modules: ${{ steps.list_modules.outputs.modules }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: List Modules
+        id: list_modules
+        run: |
+          MODULES="${{ toJson($(echo $(ls Terraform_modules))) }}

```

Comment:
what is the `"` for? i think it is there by mistake

Reply:
This is a playaround to see If I can create a matrix by reading the file directory